### ISSUE TITLE
Ability to temporarily "bless" a user

### ISSUE THREAD

**Issue by derenrich** [lucaswerkmeister/tool-speedpatrolling#8]
Great tool. I have a few ideas for expansions that I thought I would catalog in feature requests.

It's often the case that anonymous users will make a whole series of very repetitive edits in a row. They are all the same and after reviewing the first 5-10 I feel like I don't need/want to see them anymore. So I suggest a way to temporarily "bless" the user so they are treated as confirmed for me for the next e.g. 24 hours.

**Comment by lucaswerkmeister** [comment]
I’m not sure what this would do though… skip their edits just for you, even though they still need patrolling by someone else? Or automatically mark them as patrolled in the background even though nobody has actually reviewed the new edits? Neither sounds like a good idea to me, to be honest.

**Comment by derenrich** [comment]
>I’m not sure what this would do though… skip their edits just for you, even though they still need patrolling by someone else?

Yeah that's basically what I'm proposing. I guess I see patrolling edits as a solo effort to find bad edits going on now. I don't know how realistic it is to patrol all edits. And so if we are not going to get to all edits anyways then I'd like to only see edits that are likely bad. And often times there's a user making a ton of repetitive edits that I just don't want to review.
